src/Common/src/Interop/Unix/libssl/SecuritySafeHandles.cs

```diff
@@ -265,12 +276,23 @@ public override bool IsInvalid
 
         protected override bool ReleaseHandle()
         {
-            Interop.OpenSsl.FreeSslContext(_sslContext);
+            Interop.OpenSsl.Disconnect(_sslContext);
```

Comment:
Remind me why it's not SafeSslHandle's ReleaseHandle that does the Disconnect?  What if someone else is still using it?